test/src/test/kotlin/graphql/nadel/tests/TestFixture.kt

```diff
@@ -88,3 +94,20 @@ data class ExpectedException(
         )
     }
 }
+
+data class IncrementalResponse(
+    @JsonProperty("initialResponse")
```

Comment:
I'm happy to merge this as it is. But as we discussed, it's worth trying to use the actual graphql-java types here to make deserialization easier.